js/screens/wizard-screen.js

```diff
@@ -2,6 +2,7 @@ import { initWizard } from '../wizard/wizard.js';
 
 export const wizardScreen = () => {
    const wizardScreenWrapper = document.createElement('div');
```

Comment:
**suggestion:** I think this will be a good place to keep some state for your wizard, so it wraps all steps